apps/gateway/test/champions_test.exs

```diff
@@ -888,7 +883,7 @@ defmodule Gateway.Test.Champions do
     end
 
     test "leveling up the Dungeon Settlements increments the afk rewards", %{socket_tester: socket_tester} do
-      {:ok, user} = Users.register("DungeonSettlementAFKRewardsUser")
+      {:ok, user} = Users.register("TestUser")
```

Comment:
Why are you changing this names? I believe this was on purpose to differentiate users by their test purpose.

Reply:
The username has a max_length restriction of 9, all these names have a length longer than 9. For example `DungeonSettlementAFKRewardsUser` has a length of too long

Reply:
Is it ok to put this restriction for every user? Can't we do a custom validation just for "game_id=com_game_id"? This way other clients (games) can have different rules.
It's not mandatory, but we could abstract this rule beforehand.

Reply:
Hmmm yeah i can see that, since we'll share user schema with different games each one will have a different name ruling, will do!

Reply:
done!